packages/nuxt-plugin/src/index.ts

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import {
+  type UnpluginOptions,
+  createVitePlugin,
+  type VitePlugin,
+} from "unplugin";
+import {
+  type Options,
+  normalizeOptions,
+  red,
+  checkNodeVersion,
+  Output,
+} from "@codecov/bundler-plugin-core";
+import { _internal_vitePlugin } from "@codecov/vite-plugin";
```

Comment:
Sorry I didn't catch this before, but I think we should rename this to be clear that it's a bundle analysis plugin. Ditto with other usages.